components/Buttons/src/MDCButton.m

```diff
@@ -402,6 +410,9 @@ - (void)touchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event {
 #pragma mark - UIControl methods
 
 - (void)setEnabled:(BOOL)enabled {
+  if (self.enabled && !enabled) {
+    _enabledAlpha = self.alpha;
```

Comment:
IMO you should write to _enabledAlpha on setAlpha:

Reply:
Done.